### PR TITLE
JBPM-9286 UI error when parent process instance details have been rem…

### PR DESCRIPTION
**Pull request by bxf12315**
…oved

**Thank you for submitting this pull request**

**JIRA**: _(JBPM-9286)_ 

[link](https://issues.redhat.com/browse/JBPM-9286)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

paste the link(s) from GitHub here

**How to retest or run**:

* a pull request please add comment: regex **[.\*[j|J]enkins,?.\*(retest|test) this.\*]**

* a full downstream build please add comment: regex **[.*\[j|J]enkins,?.\*(execute|run|trigger|start|do) fdb.\*]**

* a compile downstream build please  add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) cdb.\*]**

* a full production downstream please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) product fdb.\*]**

* an upstream build please add comment: regex **[.\*[j|J]enkins,?.\*(execute|run|trigger|start|do) upstream.\*]**

i.e for running a full downstream build =  **Jenkins do fdb**

### CODE REVIEW

**Comment by bxf12315** [comment]
Jenkins retest it please

**Comment by bxf12315** [comment]
Jenkins retest it please

**Comment by bxf12315** [comment]
Jenkins retest it please

File: kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/AbstractKieServicesClientImpl.java [hunk 1]
```diff
@@ -210,6 +210,27 @@ public KieServerHttpRequest doOperation(String url) {
...
+        logger.debug("About to send GET request to '{}'", uri);
+        KieServerHttpRequest request = invoke(uri, new RemoteHttpOperation() {
+            @Override
+            public KieServerHttpRequest doOperation(String url) {
+                return newRequest(url).get();
+            }
+        });
+        KieServerHttpResponse response = request.response();
+
+        owner.setConversationId(response.header(KieServerConstants.KIE_CONVERSATION_ID_TYPE_HEADER));
```
Comment (nmirasch) [inline kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/AbstractKieServicesClientImpl.java:223]: @bxf12315 This first part of the method is repeated many times, I guess if we can create a private method with this and call if from makeHttpGetRequestAndCreateCustomResponse, makeHttpGetRequestAndCreateCustomResponseWithHandleNotFound and makeHttpGetRequestAndCreateRawResponse. WDYT?

Comment (bxf12315) [reply kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/AbstractKieServicesClientImpl.java:223]: ok

**Comment by bxf12315** [comment]
Jenkins retest it please

**Review by nmirasch — APPROVED** [review]
Changes look good lo me

**Comment by prashuts** [comment]
Jenkins execute full downstream build

**Comment by prashuts** [comment]
jenkins do fdb

**Comment by prashuts** [comment]
@bxf12315  FDB build got failed , can you please check on this.

**Comment by prashuts** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
Jenkins retest this

**Comment by prashuts** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
failures are not related. Affects web sockets in randome failure test

**Review by prashuts — APPROVED** [review]
Looks good to me.

**Comment by bxf12315** [comment]
Jenkins retest this please

**Pull request merged**